index.html

```diff
@@ -0,0 +1,320 @@
+          <h2>About me</h2>
+          <p class="about-me-text">
+            I am a skilled Full-Stack Web Developer who loves making pages with
+            beautiful UI. I have studied at Microverse, and I am experienced in
+            remote work.
+            <br />
+            I've also been a good helping hand to a lot of developers in
+            different communities!
+            <br />
+            if you'd like to get in contact about potential job opportunities,
+            please reach out!
+          </p>
+          <a
+            href="https://docs.google.com/document/d/1lYq0Kb50TujzJzDWGobIchRI17JMnAiZMF9OBQgNU68/edit?usp=sharing"
+            class="link"
+            target="_blank"
+            rel="noreferrer noopener"
+          >
+            Get my resume
+          </a>
+          <img class="about-me-icon" src="./media/svg/icon-2.svg" alt="Icon" />
+        </section>
+        <hr />
+        <div class="about-me-section stacks">
+          <ul>
+            <li class="languages list-title">
+              Languages
+              <ul>
+                <li>JavaScript</li>
+                <li>TypeScript</li>
+                <li>Ruby</li>
+                <li>HTML</li>
+                <li>CSS</li>
+                <li>Sass</li>
+              </ul>
+            </li>
+          </ul>
+          <ul>
+            <li class="frameworks list-title">
+              Frameworks
+              <ul>
+                <li>React & Redux</li>
+                <li>Material-UI</li>
+                <li>Jest</li>
+                <li>Bootstrap</li>
+                <li>Ruby on Rails</li>
+                <li>RSPec</li>
+              </ul>
+            </li>
+          </ul>
+          <ul>
+            <li class="skills list-title">
+              Skills
+              <ul>
+                <li>GitHub</li>
+                <li>CodePen</li>
+                <li>GitLab</li>
+                <li>Terminal</li>
+                <li>WSL</li>
+                <li>Remote Pair-Programming</li>
+                <li>Teamwork</li>
+                <li>Mentoring</li>
+                <li>Code Reviews</li>
+                <li>Mobile/Responsive Development</li>
+              </ul>
+            </li>
+          </ul>
+        </div>
+      </div>
+
+      <footer>
+        <h2 id="contact-text">I'm always interested n hearing about new projects so if you'd like to chat please get in</h2> 
+        <form action="https://formspree.io/f/meqvlald" method="POST">
+            <input class="contact-input" type="text" id="name" name="name" maxlength="30" placeholder="Full name" required>
+            <input class="contact-input" type="email" id="email" placeholder="Email address" name="_replyto" required>
+            <textarea class="contact-input" name="message" id="message"  maxlength="500"
+              placeholder="Enter text here" required></textarea>
+          <button id="submit-btn" type="submit">Get in Touch</button>
```

Comment:
- [ Optional]:  Please try to add the button interaction on the action of the user, it's a part of the project requirement but as the previous reviewer did not mention it, you can take it as optional. So do you think if we need to impress someone who is visiting our page? :thinking:  I think animations can do so even the way on hovering on something, if those kinds of interactions exist, the user can be really impressed :smiley: